docker/tensorflow-aarch64/scripts/build-openblas.sh

```diff
@@ -27,7 +27,12 @@ readonly src_repo="OpenBLAS"
 
 git clone ${src_host}/${src_repo}.git
 cd ${src_repo}
+
+if [ ${blas_cpu} == NEOVERSEV1 ] || [ ${blas_cpu} == NEOVERSEN2 ]; then
```

Comment:
This seems unnecessary complex - it might just be easier to update the `$version` of OpenBLAS that's checked out to the latest release. It looks like 0.3.19 has the patch.

Reply:
OpenBLAS patch for neoverse-v1&n2 got merged after last week only and not present in 0.3.19. This change is a staging one, can be removed once openBLAS 0.3.20 is released with this patch.

Reply:
Sorry @snadampal, my mistake, I was looking at `develop`, not the `0.3.19` tag.